mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -158,8 +158,11 @@ MMCH:
 		Recoils: no
 	Armament:
 		Weapon: 120mm
+		MuzzleSequence: muzzle
 		Recoil: 127
 		RecoilRecovery: 26
+		LocalOffset: 512, 224, 784
```

Comment:
This also looks quite far offset:
![screen shot 2016-05-28 at 18 33 38](https://cloud.githubusercontent.com/assets/167819/15628802/cf21ff5a-2502-11e6-81ea-4bbe9e77e5c0.png)